### ISSUE TITLE
Whole Cell Modeling

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepsystems#1]
Whole cell models are a very promising avenue of work in systems biology. Can whole cells models be adapted to work in drug discovery contexts?

Relevant papers:
- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3413483/
- http://www.wholecell.org/
- https://github.com/CovertLab/WholeCell
- http://www.wholecell.org/school-2017/

Weaknesses in current work:
- The code base for the Covert lab's whole cell model is in Matlab. Would need to create a python port.
- Code is specific for M. Genitalium. Would need to create a system capable of being applied to different tumor cell lines or the like.

**Comment by vsomnath** [comment]
Whole cell models could be adapted to work in drug discovery contexts, but it might be easier to start with building these whole cell model frameworks for disease causing agents (for example Mycoplasmas Tuberculosis), before we go to tumor cell lines and the like.

Whole Cell Modelling so far, looks to have been applied to organisms that have been well studied and those which have a lesser number of components. For tumor cell lines, getting the right data is harder, the cellular system is much more complex and there is much yet to be discovered.

What do you think?

**Comment by rbharath** [comment]
Yes, definitely agree. Making whole cell models for human cell lines will involve significant additional work and starting simpler is a good plan.
